### PR TITLE
feat(funke): initialized new wallet and agent setup

### PR DESCRIPTION
**Pull request by berendsliedrecht**
Signed-off-by: Berend Sliedrecht <sliedrecht@berend.io>

### CODE REVIEW

**Review by TimoGlastra — COMMENTED** [review]
We can probably move more out of the app, now is the easiest time to do it i think as there's no duplication yet

**Comment by berendsliedrecht** [comment]
> We can probably move more out of the app, now is the easiest time to do it i think as there's no duplication yet

What do you mean exactly?

File: apps/funke/app/_layout.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import { initializeAppAgent } from '.'
+
+void SplashScreen.preventAutoHideAsync()
+
+export const unstable_settings = {
+  // Ensure any route can link back to `/`
+  initialRouteName: 'index',
+}
+
+export default function HomeLayout() {
```
Comment (TimoGlastra) [inline apps/funke/app/_layout.tsx:25]: Still feel we can improve on duplication here (e.g. with fonts, initialization, hooks). But let's merge now and we can look at it later if needed. Good chance we can all throw it away in a few months anyway 🤷

File: apps/funke/app/utils/DeeplinkHandler.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
+import { useCallback } from 'react'
 import type { ReactNode } from 'react'
```
Comment (TimoGlastra) [inline apps/funke/app/utils/DeeplinkHandler.tsx:2]: This one we can also move?

File: apps/paradym/app/utils/DeeplinkHandler.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import { useNavigation } from 'expo-router'
+import { useEffect, useState } from 'react'
+
+interface DeeplinkHandlerProps {
+  children: ReactNode
+}
+
+const deeplinkSchemes = Object.values(InvitationQrTypes)
+
+export const DeeplinkHandler = ({ children }: DeeplinkHandlerProps) => {
```
Comment (TimoGlastra) [inline apps/paradym/app/utils/DeeplinkHandler.tsx:18]: Duplicated between apps

File: packages/agent/package.json [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
-    "credo-ts-didweb-anoncreds": "0.0.1-alpha.13",
-    "query-string": "^8.1.0",
-    "rxjs": "^7.8.1"
-  },
-  "peerDependencies": {
-    "@hyperledger/anoncreds-react-native": "*",
-    "@hyperledger/aries-askar-react-native": "*",
-    "@hyperledger/indy-vdr-react-native": "*"
-  }
+	"name": "@package/agent",
```
Comment (TimoGlastra) [inline packages/agent/package.json:2]: Why @package over @internal?

Fine but never seen @package before

**Review by TimoGlastra — APPROVED** [review]

File: packages/agent/package.json (hunk 4 above)
Comment (berendsliedrecht) [reply packages/agent/package.json:2]: `@package` is what we use for idcrypt.

**Pull request merged**